### ISSUE TITLE
400 bad request

### ISSUE THREAD

**Issue by cedmax** [thesp0nge/wordstress#5]
I tried the version 0.40.0 but I receive the following error

```
14:44:55 [*] wordstress v0.40.0 is starting up
14:44:55 [$] wordstress: storing results to /Users/marco/wordstress/mydomain.net_80_wordstress/20150225
14:44:55 [$] wordstress: scanning http://mydomain.net/wordstress
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/net/http/response.rb:119:in `error!': 400 "Bad Request" (Net::HTTPServerException)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/net/http/response.rb:128:in `value'
	from /Library/Ruby/Gems/2.0.0/gems/wordstress-0.40.0/lib/wordstress/site.rb:356:in `get_http'
	from /Library/Ruby/Gems/2.0.0/gems/wordstress-0.40.0/lib/wordstress/site.rb:352:in `get_http'
	from /Library/Ruby/Gems/2.0.0/gems/wordstress-0.40.0/lib/wordstress/site.rb:352:in `get_http'
	from /Library/Ruby/Gems/2.0.0/gems/wordstress-0.40.0/lib/wordstress/site.rb:155:in `get'
	from /Library/Ruby/Gems/2.0.0/gems/wordstress-0.40.0/lib/wordstress/site.rb:35:in `initialize'
	from /Library/Ruby/Gems/2.0.0/gems/wordstress-0.40.0/bin/wordstress:89:in `new'
	from /Library/Ruby/Gems/2.0.0/gems/wordstress-0.40.0/bin/wordstress:89:in `<top (required)>'
	from /usr/bin/wordstress:23:in `load'
	from /usr/bin/wordstress:23:in `<main>'
```

could this be related to the fact that my wordpress is in https?
Could this be related to the php error I raised -> https://github.com/thesp0nge/wordstress/issues/4 ?

**Comment by markri** [comment]
Experiencing same problem with 0.50.0
Also having my site under HTTPS and running under nginx

**Comment by thesp0nge** [comment]
Should be fixed by @markri pull request #8 .

Soon I'll release a new version.
Thank you guys
